.stylelintrc.json

```diff
@@ -9,6 +8,7 @@
   ],
   "rules": {
     "order/properties-alphabetical-order": true,
-    "no-descending-specificity": null
+    "no-descending-specificity": null,
+    "scss/at-import-partial-extension": "always"
```

Comment:
Isn't this the default from the `stylelint-config-recommended-scss` or am I mixing that up with `scss/at-extend-no-missing-placeholder`?

Reply:
as far as I see - default value from `stylelint-config-recommended-scss` is `never` but we are using extensions everywhere, so I added this config